### ISSUE TITLE
support border padding when perform convolution?

### ISSUE THREAD

**Issue by dzhwinter** [fchollet/keras#412]
I'm new to keras. would someone here figure it out that how to perform border padding with value such as 1,0 ? Did it built-in support in keras?

**Comment by pranv** [comment]
You can do zero padding with `ZeroPadding2D` layer. It's with other convolution layers.
